README.md

```diff
@@ -24,6 +24,11 @@ at the [Beta](#versioning) quality level:
 
 - [Google Cloud Bigtable](google/cloud/bigtable#google-cloud-bigtable)
 
+This library supports the following Google Cloud Platform services with clients
+at the [Alpha](#versioning) quality level:
+
+- [Google Cloud Storage](google/cloud/storage#google-cloud-storage)
```

Comment:
For easier reference and easier future copy-pasting, you can just link to `#readme` which is a standard built-in tag for the beginning of the README, so you don't need to carefully emulate the actual heading.

If you fix this, consider also fixing the Cloud Bigtable link above similarly. Up to you.